lib/JSONAPISerializer.js

```diff
@@ -571,6 +571,10 @@ module.exports = class JSONAPISerializer {
       deserializedData.meta = data.meta;
     }
 
+    if (options.afterDeserialize) {
+      return options.afterDeserialize(deserializedData, options);
```

Comment:
Do you think `options` should be needed as a second argument ?

Reply:
Originally I thought maybe someone wanted to have access to `id`, `whitelist` or `blacklist`.

But now I think it's better to avoid technical debt. If someone wants it, they can open an issue.